js/react_native/lib/binding.ts

```diff
@@ -64,12 +71,41 @@ export declare namespace Binding {
 
   interface InferenceSession {
     loadModel(modelPath: string, options: SessionOptions): Promise<ModelLoadInfoType>;
-    loadModelFromBase64EncodedBuffer?(buffer: string, options: SessionOptions): Promise<ModelLoadInfoType>;
+    loadModelFromBlob?(blob: JSIBlob, options: SessionOptions): Promise<ModelLoadInfoType>;
     dispose(key: string): Promise<void>;
     run(key: string, feeds: FeedsType, fetches: FetchesType, options: RunOptions): Promise<ReturnType>;
   }
 }
 
 // export native binding
-const {Onnxruntime} = NativeModules;
+const {Onnxruntime, OnnxruntimeJSIHelper} = NativeModules;
 export const binding = Onnxruntime as Binding.InferenceSession;
+
+// install JSI helper global functions
+OnnxruntimeJSIHelper.install();
+
+declare global {
+  // eslint-disable-next-line no-var
+  var jsiOnnxruntimeStoreArrayBuffer: ((buffer: ArrayBuffer) => JSIBlob)|undefined;
+  // eslint-disable-next-line no-var
+  var jsiOnnxruntimeResolveArrayBuffer: ((blob: JSIBlob) => ArrayBuffer)|undefined;
```

Comment:
I disable `no-var` for ESLint because it required to use `var` for declare global in TS.